src/components/Catalog.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+import CatalogSection from './CatalogSection'
+
+export default class Catalog extends Component {
+    constructor(){
+        super()
+        this.state={
+            searchTerm: "",
+        }
+    }
+    rentMovie = (id, result ) =>{
+        this.props.rentMovie(id, result)
+    }
+    handleSearch = (event) =>{
```

Comment:
can also write it like
`handleSearch = ({target}) => this.setState({searchTerm: target.value})`